spinnaker/spinnaker_system/google_front50_test.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import sys
+
+import citest.gcp_testing as gcp
+import citest.json_contract as jc
+import citest.json_predicate as jp
+import citest.service_testing as st
+
+# Spinnaker modules.
+import spinnaker_testing as sk
+import spinnaker_testing.front50 as front50
+
+
+class GoogleFront50TestScenario(sk.SpinnakerTestScenario):
+  @classmethod
+  def new_agent(cls, bindings):
+    """Implements citest.service_testing.AgentTestScenario.new_agent."""
+    return front50.new_agent(bindings)
+
+  @classmethod
+  def initArgumentParser(cls, parser, defaults=None):
+    """Initialize command line argument parser.
+
+    Args:
+      parser: argparse.ArgumentParser
+    """
+    super(GoogleFront50TestScenario, cls).initArgumentParser(
+        parser, defaults=defaults)
+
+  def _do_init_bindings(self):
+    """Hook to initialize custom test bindings so journaling is scoped."""
+    config = self.agent.deployed_config
+    enabled = config.get('spinnaker.gcs.enabled', False)
+    if not enabled:
+      raise ValueError('spinnaker.gcs.enabled is not True')
+
+    bucket = config['spinnaker.gcs.bucket']
```

Comment:
Is it not acceptable for people to specify the bucket with a [`gs://` suffix](https://github.com/google/citest/pull/159/files#diff-5a5b021e06e0c7fc8ea1360cfd485c83R58)?